Monika After Story/game/script-farewells.rpy

```diff
@@ -1116,7 +1170,25 @@ label bye_going_somewhere_normalplus_flow_aff_check:
         #   and will ask u to wait for her to get ready
         m 1sub "Really?"
         m 1hua "Yay!"
-        m 1ekbfa "I wonder where you'll take me today..."
+        if mas_isMoniLove(higher=True):
```

Comment:
This block is inside an `else` that triggers if the player doesn't meet the love check, so this won't execute either.